CONTRIBUTING.md

```diff
@@ -80,6 +80,13 @@ For example, a new change to the `miden-node-store` crate might have the followi
 
 You can find more information about the `cargo make` commands in the [Makefile](Makefile.toml)
 
+### Testing
+- After writing code different types of tests (unit, integration, end-to-end) are required to make sure that the correct behavior has been achieved and that no bugs have been introduced. You can run tests using the following command:
```

Comment:
We don't specifically need to but it's the format that has been using throughout the file, hence I copied it.

We could change it but then it would meed reformatting the whole file to make styling consistent and make sure that code blocks are shown correctly. ( I can't just remove the bullet points, it would break the code blocks ).

Reply:
I think in other sections the bullet points make sense as there are multiple bullet points. In this section, this is not really a bullet point but a single paragraph - so, I think removing bullet point from here is fine (i.e., we don't need to reformat anything else).